data/successors/successor 1 prologue.txt

```diff
@@ -0,0 +1,1042 @@
+			choice
+				`	"I accept."`
+					goto accepted
+				`	"You want me to do your errands? I won't!"`
+					goto rejected
+			label rude
+				`	"Thine appreciation, or thy failure t'appreciate, are not the sort of things that would concern us in this hall. Thou may'st be an alien, still swaddled in thine ignorance, and yet this makes thee even more a guest upon our worlds and, a hm... consumer of our grace. Thy compliance is expected; in return, our generosity.`
+			choice
+				`	"I apologize."`
+				`	"I have no desire to be your servant."`
+					goto rejected
+			`	"Thine apology is noted."`
+				goto task
+			label accepted
+			action
+				log "Met with the leaders of Successor High House Sioeora and was instructed to retrieve an animal from the planet Raaqa-Uur-Kaav. Was promised translation equipment as a reward. Accepted their offer."
+			`	"Thou show'st wisdom in thy choice. Even now, thy ship is being prepared within the spaceport o'this estate. Thou should'st travel there when thine audience is finished, and there begin thy noble task with haste." The scion pauses for a moment before continuing. "But prithee note, dear Captain..." Sieasej shivers with color as the scion speaks the next sentence, "that we do not entrust our citizen to thee 'thout consideration. If harm befalls the speaker of these words whilst under thy protection, or if thou fail'st t'return them here in thirty days, we will consider thee an enemy and levy upon thee every harm that we can muster."`
+				goto end
+			label rejected
+			action
+				log "Met with the leaders of Successor High House Sioeora and was instructed to retrieve an animal from the planet Raaqa-Uur-Kaav. Was promised translation equipment as a reward. Did not accept."
+				set "rejected: Successors: First Contact 2"
+				"reputation: House Sioeora" <?= 0
+			`	A ripple of color runs up the body of Sieasej and several of the other Successors. The scion gives their response slowly. "Then our act of grace today shall be t'allow thee to depart this noble place still possessed o'thy freedom and belongings. Thou shalt receive no further aid nor comfort from this House, and shalt be banished at the pleasure of its scions from whatever worlds lie under our dominion."`
+			`	Sieasej and the scions turn their backs to you, and a line of soldiers converge from the walls to form a line separating you from them. The lead officer who escorted you inside motions with their weapon for you to leave.`
+			`	As you exit, the door closes behind you, reforming into solid metal.`
+			label end
+				decline
+
+mission "Successors: First Contact 3"
+	priority
+	name "Capture an animal for House Sioeora"
+	description `The scions of House Sioeora have instructed you to visit the planet <planet stopovers> and capture a "Kijra-riitaa" to safely return to them at <planet>. Return by <date> or they will consider you to have kidnapped Sieasej.`
+	source "Staja-Kella-Oa"
+	to offer
+		has "Successors: First Contact 2: offered"
```

Comment:
```suggestion
		has "event: successors: first contact wait"
```
This should fix the issue tibe has encountered with not being offered this mission until taking off and landing again.
I'm also not sure that this mission needs to or should be "priority".